activejob/lib/active_job/core.rb

```diff
@@ -115,7 +115,8 @@ def serialize
         "exception_executions" => exception_executions,
         "locale"     => I18n.locale.to_s,
         "timezone"   => timezone,
-        "enqueued_at" => Time.now.utc.iso8601(9)
+        "enqueued_at" => Time.now.utc.iso8601(9),
+        "scheduled_at" => scheduled_at_time ? scheduled_at_time.utc.iso8601(9) : nil,
```

Comment:
This is possibly parsing the time twice in the case of a string. Should we memoize the value or set in the initializer?

Reply:
I made the `scheduled_at=` setter do the coercion to `@_scheduled_at_time` so it only happens once. That also looked gnarly enough that I went ahead and added a deprecation warning about using a numeric so that we could remove it in the future:

https://github.com/rails/rails/blob/5e9e50ff23db172e4fc0d6826b3036b613069a7c/activejob/lib/active_job/core.rb#L177-L187